Sources/SwiftDocC/Infrastructure/Topic Graph/AutomaticCuration.swift

```diff
@@ -125,6 +132,19 @@ public struct AutomaticCuration {
         
         let parentReference = canonicalPath.last!
         
+        func filterReferences(_ references: [ResolvedTopicReference]) throws -> [ResolvedTopicReference] {
```

Comment:
Nit: Is there a more specific name we can use to make this more readable at the call-site?

```suggestion
        func referencesAvailableInCurrentTrait(in references: [ResolvedTopicReference]) throws -> [ResolvedTopicReference] {
```

Reply:
It also filters out the current reference from the task group, though. We could also consider making this function only handle the trait-based filtering, but the logic to filter out the current reference would then be duplicated.